api/tests/unit/application/target-profiles/index_test.js

```diff
@@ -922,4 +922,60 @@ describe('Unit | Application | Target Profiles | Routes', function () {
       });
     });
   });
+
+  describe('POST /api/admin/target-profiles/{targetProfileId}/copy', function () {
+    describe('When user has role SUPER_ADMIN', function () {
+      it('Should return a response with 200 status code', async function () {
+        // given
+        const method = 'POST';
+        const url = '/api/admin/target-profiles/123/copy';
+
+        sinon
+          .stub(securityPreHandlers, 'hasAtLeastOneAccessOf')
+          .withArgs([
+            securityPreHandlers.checkAdminMemberHasRoleSuperAdmin,
+            securityPreHandlers.checkAdminMemberHasRoleMetier,
+          ])
+          .callsFake(() => (request, h) => h.response(true));
+        sinon.stub(targetProfileController, 'copyTargetProfile').callsFake((request, h) => h.response('ok').code(200));
+        const httpTestServer = new HttpTestServer();
+        await httpTestServer.register(moduleUnderTest);
+
+        // when
+        const { statusCode } = await httpTestServer.request(method, url);
+
+        // then
+        expect(statusCode).to.equal(200);
```

Comment:
**suggestion** : 

c'est une chose que l'on faisait beaucoup ça de tester le code retour que l'on a mock (à tort). mais au final ce que l'on souhaite tester c'est que le controller est appelé

cas ok `expect(targetProfileController.called).to.be.true`
cas ko (🍫) `expect(targetProfileController.notCalled).to.be.true`